src/main.js

```diff
@@ -29,16 +29,21 @@ require.config({
     paths: {
         "text"              : "thirdparty/text/text",
         "i18n"              : "thirdparty/i18n/i18n",
-        "react"             : "thirdparty/react",
 
         // The file system implementation. Change this value to use different
         // implementations (e.g. cloud-based storage).
         "fileSystemImpl"    : "filesystem/impls/appshell/AppshellFileSystem"
     },
+    packages: [
```

Comment:
We should probably add `preact` as an `node_module` instead of inlining it to the `thirdparty` folder as we have been doing with new dependencies.  (related to #12006)